### DISCUSSION TITLE
WebRequest Error Timeout für Vorgang überschritten

### DISCUSSION THREAD

**Discussion by Christian-4711** [Claus-E/teams2tasmota#7] [Q&A]
Hallo zusammen!

Das wichtigste zuerst: Das ist ein super tolles Projekt. Vielen Dank dafür!

Ich habe am Samstag meine Tasmota-Glühlampe bekommen und bin somit seit heute im echten Testbetrieb.
Die Lampe schaltet hervorragend von der ein in die andere Farbe.
Ab und zu, ich glaube wenn der Status in Teams länger nicht gewechselt wird, ändert sich die Farbe nicht mehr und ich muss das Executable beenden und neu starten.
Vorher sieht man die Meldung "WebRequest Error Timeout für Vorgang überschritten" im Teams2Tasmota Fenster:

![grafik](https://user-images.githubusercontent.com/111051502/236793583-6e4498e5-3c1c-45bc-8a4d-931345c7b3f8.png)

Woran kann das liegen? 
Was kann ich tun?
Ist das eher ne Tasmota-Einstellung?

Vielen Dank im Voraus und viele Grüße,
Christian

**Comment by Claus-E** [comment]
Hallo Christian, ich würde vermuten, dass es was mit dem WLAN Empfang am Standort der Lampe zu tun hat. Versuche doch Mal die Weboberfläche zu erreichen, wenn der Timout angezeigt wird.

Gruß Claus

**Comment by Christian-4711** [comment]
Hallo Claus,

danke für den Tipp.
Hab´s gerade ausprobiert....aber das scheint nicht das Problem zu sein. Die Weboberfläche ist zu dem Zeitpunkt erreichbar.
Wenn ich den Teams Status danach manuell auf irgendwas ändere, schaltet die Lampe auch nicht um und in Deinem "Executable Fenster" taucht in der entsprechenden Zeile dann auch diese Webrequest Fehlermeldung auf.
Ich muss es schließen und neu starten. 
Als Administrator ausführen macht keinen Unterschied.

Mmmhhh, ich forsche mal weiter...
 
Gruß,
Christian
